ignite/pkg/cosmosaccount/cosmosaccount.go

```diff
@@ -133,12 +139,18 @@ func (a Account) Address(accPrefix string) string {
 		accPrefix = AccountPrefixCosmos
 	}
 
-	return toBench32(accPrefix, a.Info.GetPubKey().Address())
+	// TODO: handle error
+	pk, _ := a.Record.GetPubKey()
```

Comment:
Why not panicking here until the error is handled?

Also, we should track TODOs in new Github issues if we deem it can be worked on later